winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+var options = new TreeGridExcelExportOptions();
+options.ExportMode = ExportMode.Text;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Export without outlines	
+
+By default, all the children in the TreeGrid will be exported based on the state in TreeGrid. You can disable the outlines in Excel by setting the `ShowOutlines` property to false in `TreeGridExcelExportOptions` class.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.ShowOutlines = false;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-export-to-excel.png" alt="WinUI TreeGrid Data Exporting to Excel without Outlines" width="100%" Height="Auto"/>
+
+### Exclude columns while exporting
+
+By default, all the columns (including hidden columns) in the TreeGrid will be exported to Excel. To exclude some columns, use `ExcludedColumns` field in `TreeGridExcelExportOptions` class.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.ExcludedColumns.Add("FirstName");
+options.ExcludedColumns.Add("LastName");
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+Here, the columns having FirstName and LastName as MappingName are excluded when exporting.
+
+### Excel version
+
+While exporting to Excel, specify the excel version by using the `ExcelVersion` property.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting stacked headers
+
+Export the stacked headers to excel by setting the `CanExportStackedHeaders` property to `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.CanExportStackedHeaders = true;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting merged cells
+
+Export the merged cells to excel by setting the `CanExportMergedCells` property as `true`.

```

Comment:
Export the merged cells to excel by setting the `CanExportMergedCells` property as `true.`

Reply:
content updated